### ISSUE TITLE
Basic_WebSkeletonApp example is missing bootstrap and jquery resources

### ISSUE THREAD

**Issue by robotiko** [SmingHub/SmingRTOS#90]
Missing bootstrap and jquery resources

**Comment by hreintke** [comment]
@avr39-ripe 

I looked once more and now in detail to your webskeleton example and see the following as possible enhancements.
- update the use of static buffer and staticjsonbuffer to dynamic
- Rename all items which have "therm" in it to a more generic term
- Make the name of the configfile adaptable with an environment variable
- Make the loadconfig and saveconfig class functions 
- Rename the configuration.cpp to therm.cpp -> ClassName.h & ClassName.cpp are accepted standards
- Instantiate the ActiveConfig within the application -> keep the class definition separated form instantiation
- Check for valid configuration data from config file within loadconfig()
- The rename the webserver.cpp as the "application logic" is mainly that
- Clarify the process within onConfiguration.
- the root.prettyprint goes to serial (cannot switch off)
- test complete for valid json input, not only stassid
- split between "setting/saving" config and activating config
- Can I get the stassid/paswsword by just knowing the right url ?
- Make real use of the new WifiEvents and wait for network before starting servers etc
- Not sure but what is the behavior of the application when the initial ssid/pwd just not connect ?

**Comment by hreintke** [comment]
@avr39-ripe : 
Will add to this issue when I see/more other if that is OK.
- Updated the `SPIFF_SIZE     = 196608` to a value which is actually needed to prevent long during flashes
- What is the rationale for these settings ?

```
    //SET higher CPU freq & disable wifi sleep
    system_update_cpu_freq(SYS_CPU_160MHZ);
    wifi_set_sleep_type(NONE_SLEEP_T);
```

**Comment by avr39-ripe** [comment]
@hreintke YES! indeed!!! anybody is welcome to comment this app and suggest better aproaches, or just rise issues.. thank you very much for good critic!!

**Comment by avr39-ripe** [comment]
//SET higher CPU freq & disable wifi sleep
    system_update_cpu_freq(SYS_CPU_160MHZ);
    wifi_set_sleep_type(NONE_SLEEP_T);

faster CPU :) less time to do things, second is more doubtfull, found somewhere as to enhance wifi stability.. will remove it as unconfirmed.
